tools/topology/development/sof-tgl-nocodec-ci.m4

```diff
@@ -242,7 +242,7 @@ DAI_CONFIG(DMIC, 0, 6, NoCodec-6,
 	   dnl DMIC_CONFIG(driver_version, clk_min, clk_mac, duty_min, duty_max,
 	   dnl		   sample_rate, fifo word length, unmute time, type,
 	   dnl		   dai_index, pdm controller config)
-	   DMIC_CONFIG(1, 500000, 4800000, 40, 60, 48000,
+	   DMIC_CONFIG(1, 2400000, 4800000, 40, 60, 48000,
```

Comment:
@singalsu can you please rebase this PR? I think this patch may no longer be needed after PIerre's changes

Reply:
Sure, I start fixing today. I didn't know if it's needed.

Reply:
Done!

Reply:
@ranj063 are we good now ? @plbossart changes have been merged and I dont see any conflicts.